charts/zservice/charts/redis/values.yaml

```diff
@@ -0,0 +1,18 @@
+redis:
+  usePassword: true
+  existingSecret: redis
+
+  image:
+    tag: "6.0"
+
+  cluster:
+    enabled: false
+
+  # set maxmemory + maxmemory-policy in order to use redis as an lru cache. See
+  # https://redis.io/topics/lru-cache
+  maxmemory: 1gb
+  maxmemoryPolicy: volatile-lfu
+
+  configmap: |
+    maxmemory {{ .Values.maxmemory }}
+    maxmemory-policy {{ .Values.maxmemoryPolicy }}
```

Comment:
This is cool -- I did not know that it was possible to reference .Values from within values.yaml.

Reply:
Yeah its kind of crazy.